desk/ted/commit.hoon

```diff
@@ -0,0 +1,45 @@
+/-  spider
+/-  *herd
+/+  sio=strandio
```

Comment:
either import as `*strandio` or `=strandio` - both are pretty standard across `%base` so it's a little confusing when they're aliased differently.